### DISCUSSION TITLE
Is the .exe version supposed to work with HIP on Windows atm, or do I need to build from source?

### DISCUSSION THREAD

**Discussion by one-lithe-rune** [LostRuins/koboldcpp#417] [Q&A] [answered]
I get the following error using the .exe version, with the `--usecublas` flag with the ROCm 5.5 SDK installed on Windows.

```
CUDA error 35 at C:\Users\HCH\Desktop\koboldcpp\ggml-cuda.cu:5062: CUDA driver version is insufficient for CUDA runtime version
```

Before I try building from source is that expected atm, or should I put in a issue for it?

**Comment by LostRuins** [comment, marked as answer]
No, the exe version is CUDA only. HIP has to be manually compiled

**Comment by ccbadd** [comment]
Any instructions for the compile under windows?

**Reply by dialup-coconut** [reply]
I would like to request this also.

**Reply by LostRuins** [reply]
@yellowrosecx is working on a prebuilt binary specially for AMD users, so stay tuned.

**Reply by ccbadd** [reply]
I guess it's done:
https://github.com/YellowRoseCx/koboldcpp-rocm/releases/tag/Windows-v1.43-ROCm

**Reply by one-lithe-rune** [reply]
Scruffy gist of what I'm using to build @YellowRoseCx's fork FWIW: 

https://gist.github.com/one-lithe-rune/4dcaaf51d19158ce1e9dfd2328607963

...initially made it before the very latest changes went in to CMakeLists.txt for the prebuilt, so may have redundant stuff. Does still work for me now I've tweaked it for the .dll name change.
